api/main.py

```diff
@@ -118,6 +117,25 @@ async def get_user(user: User = Depends(get_current_user)):
     return user
 
 
+async def authorize_user(node_id: str, user: User = Depends(get_current_user)):
+    """Return the user if active, authenticated, and authorized"""
+    if not user.active:
+        raise HTTPException(status_code=400, detail="Inactive user")
+
+    # Only the user that created the node or any other user from the permitted
+    # user groups will be allowed to update the node
+    node_from_id = await db.find_by_id(Node, node_id)
+    if node_from_id.user:
```

Comment:
Shouldn't all nodes have an owner?

Reply:
This check is due to already existing nodes do not have ownership details.
Maybe we can create a user and assign it to all the existing nodes?

Reply:
Yes I think all existing nodes should be updated with `admin` user or something like that.  We can also delete old nodes as this is the staging database.

Reply:
Yes, I think deleting nodes would be easier.